traffic_ops/v4-client/cdn_notifications.go

```diff
@@ -0,0 +1,49 @@
+/*
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package client
+
+import (
+	"fmt"
+	"github.com/apache/trafficcontrol/lib/go-tc"
+	"net/http"
+)
+
+const (
+	API_CDN_NOTIFICATIONS = apiBase + "/cdn_notifications"
```

Comment:
Can you add a GoDoc to this exported symbol? Or just un-export it, I don't think anything needs it. Although I know (almost) all other client files use this casing for their request paths.

Reply:
so api_cdn_notifications instead? sorry, i'm used to constants being uppercase, exported (public) or not (private).

Reply:
Yeah, me too, that's why I casefolded them all to that. But they *will* be exported if we keep doing this, whether that's what we want or not. I think snake_case is ugly, so I'd do `apiCDNNotification`, but `go fmt` won't care either way, so neither do I.

Reply:
FYI the client changes here will need to be updated once https://github.com/apache/trafficcontrol/pull/5552 is merged. That PR replaces all the snake casing w/ more "Go idiomatic" casing -- e.g. `APICDNNotifications`.

Reply:
fixing. going to leave out godoc as nowhere else does that.